content/id/docs/tasks/inject-data-application/distribute-credentials-secure.md

```diff
@@ -0,0 +1,253 @@
+
+   ```shell
+   kubectl create secret generic backend-user --from-literal=backend-username='backend-admin'
+   ```
+
+*  Tentukan nilai `backend-username` yang didefinisikan di Secret ke variabel lingkungan `SECRET_USERNAME` di dalam spesifikasi Pod.
+
+   {{< codenew file="pods/inject/pod-single-secret-env-variable.yaml" >}}
+
+*  Membuat Pod:
+
+   ```shell
+   kubectl create -f https://k8s.io/examples/pods/inject/pod-single-secret-env-variable.yaml
+   ```
+
+*  Di dalam _shell_ kamu, tampilkan konten dari variabel lingkungan `SECRET_USERNAME` dari Container
+
+   ```shell
+   kubectl exec -i -t env-single-secret -- /bin/sh -c 'echo $SECRET_USERNAME'
+   ```
+
+   Hasil keluarannya
+   ```
+   backend-admin
+   ```
+
+### Mendefinisikan variabel lingkungan Container dengan data dari banyak Secret
+
+*  Seperti contoh sebelumnya, buat Secret terlebih dahulu.
+
+   ```shell
+   kubectl create secret generic backend-user --from-literal=backend-username='backend-admin'
+   kubectl create secret generic db-user --from-literal=db-username='db-admin'
+   ```
+
+*  Definisikan variabel lingkungan di dalam spesifikasi Pod.
+
+   {{< codenew file="pods/inject/pod-multiple-secret-env-variable.yaml" >}}
+
+*  Membuat Pod:
+
+   ```shell
+   kubectl create -f https://k8s.io/examples/pods/inject/pod-multiple-secret-env-variable.yaml
+   ```
+
+*  Di dalam _shell_ kamu, tampilkan konten dari variabel lingkungan Container
+
+   ```shell
+   kubectl exec -i -t envvars-multiple-secrets -- /bin/sh -c 'env | grep _USERNAME'
+   ```
+   Hasil keluarannya
+   ```
+   DB_USERNAME=db-admin
+   BACKEND_USERNAME=backend-admin
+   ```
+
+
+## Mengonfigurasi semua pasangan _key-value_ di dalam Secret sebagai variabel lingkungan Container
+
+{{< note >}}
+Fitur ini tersedia mulai dari Kubernetes v1.6 dan lebih baru.
+{{< /note >}}
+
+*  Membuat Secret yang berisi banyak pasangan _key-value_
+
+   ```shell
+   kubectl create secret generic test-secret --from-literal=username='my-app' --from-literal=password='39528$vdg7Jb'
+   ```
+
+*  Gunakan envFrom untuk mendefinisikan semua data Secret sebagai variabel lingkungan Container. _Key_ dari Secret akan mennjadi nama variabel lingkungan di dalam Pod.
+
+    {{< codenew file="pods/inject/pod-secret-envFrom.yaml" >}}
+
+*  Membuat Pod:
+
+   ```shell
+   kubectl create -f https://k8s.io/examples/pods/inject/pod-secret-envFrom.yaml
+   ```
+
+* Di dalam _shell_ kamu, tampilkan variabel lingkungan Container `username` dan `password`
+
+  ```shell
+  kubectl exec -i -t envfrom-secret -- /bin/sh -c 'echo "username: $username\npassword: $password\n"'
+  ```
+
+  Hasil keluarannya
+  ```
+  username: my-app
+  password: 39528$vdg7Jb
+  ```
+
+### Referensi
+
+* [Secret](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#secret-v1-core)
+* [Volume](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#volume-v1-core)
+* [Pod](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#pod-v1-core)
```

Comment:
Need to check the reference link because it is invalid link.

Reply:
Looks like it's fine and no error found from deploped preview page

Reply:
Ok. We can check later after merged.